### DISCUSSION TITLE
Features version of Elastic Stack

### DISCUSSION THREAD

**Discussion by ow111** [Security-Onion-Solutions/securityonion#2644] [Unsupported Versions] [answered]
After running so-features-enable we moved to the Features version of elastic stack, Wondering if this license lasts for a certain amount of time (30 days?) or will a purchase of an elastic license be necessary

**Comment by TOoSmOotH** [comment, marked as answer]
so-features-enable installs features which includes the basic license which does not expire. If you enable paid features then that would depend on the term of that license with Elastic.
